cfme/tests/intelligence/reports/test_import_export_reports_widgets.py

```diff
@@ -145,3 +140,60 @@ def test_export_report(appliance, report_file):
         report_fields=["Archived", "Autostart", "Boot Time"],
     )
     collection.export_report(report.menu_name)
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize("overwrite", [True, False], ids=("overwrite", "replace"))
+def test_import_duplicate_report(appliance, report, overwrite):
+    """
+    This case tests appliance behavior when a duplicate report is imported.
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/4h
+    """
+    collection = appliance.collections.reports
+    file_path = yaml_path("import_report")
+    collection.import_report(file_path)
+
+    view = collection.create_view(ImportExportCustomReportsView)
+    view.flash.assert_message("Imported Report: ", partial=True)
+
+    collection.import_report(file_path, overwrite=overwrite)
+
+    if overwrite:
+        view.flash.assert_message("Replaced Report: [{}]".format(report.menu_name))
+    else:
+        view.flash.assert_message(
+            "Skipping Report (already in DB): [{}]".format(report.menu_name)
+        )
+
+
+@pytest.mark.parametrize("yaml_name", ["invalid_yaml", "invalid_report"])
+def test_reports_invalid_file(appliance, yaml_name):
+    """
+    Polarion:
```

Comment:
optional: Its look like a negative test. If yes, you can add it in polarion docstring